packages/ipfs-core/src/index.js

```diff
@@ -13,6 +13,10 @@ const multihash = multihashing.multihash
 const CID = require('cids')
 const IPFS = require('./components')
```

Comment:
Seems like it might be simpler to just do this ?

```suggestion
const { create } = require('./components')

/**
 * @typedef {import(‘./components’)} IPFS
 */
```

Reply:
Yes